compiler/src/dotty/tools/dotc/reporting/Message.scala

```diff
@@ -2,6 +2,9 @@ package dotty.tools
 package dotc
 package reporting
 
+import core.Contexts._, core.Decorators._, core.Mode
```

Comment:
While we're at it, perhaps new wildcards should be `*` or `given`. I see a recent Odersky import is Scala 3.

Reply:
I find it somewhat amusing that I slipped up there considering I used old-style imports as one of the tests  :)